### DISCUSSION TITLE
Extra Utilities 2 Screen alternative image hosts

### DISCUSSION THREAD

**Discussion by Nivid-IT** [ACGaming/UniversalTweaks#475] [Tweak Suggestions]
Extra Utilities 2 (at least in version 1.12.2) screens are no longer capable of fetching and rendering images hosted on Imgur.
This was extensively tested in singleplayer and multiplayer instances, with a variety of images well below 255kb in size, and any form of rate-limiting has also been ruled out as far as my testing goes. I believe changes made by Imgur are the culprit.

Extra Utilities 2 itself is no longer developed, and its screens and the code behind it are entirely hardcoded around using Imgur.
It would be great to see a workaround that allows for EU2 screens to use custom hosts much like MrCrayFish's TVs allow you to define/whitelist hosts from which to fetch images.

**Comment by xJon** [comment]
Imgur support is resolved with https://www.curseforge.com/minecraft/mc-mods/xu2-patcher
